src/types/onyx/Domain.ts

```diff
@@ -28,6 +28,68 @@ type Domain = OnyxCommon.OnyxValueWithOfflineFeedback<{
 
     /** Errors that occurred when fetching validation code */
     validateCodeError?: OnyxCommon.Errors;
+
+    /** Domain-related settings */
+    settings?: {
+        /** Whether setting SAML enabled/disabled setting is in progress */
+        isSamlEnabledLoading?: boolean;
+
+        /** Whether setting SAML enabled/disabled setting has failed and why */
+        samlEnabledError?: OnyxCommon.Errors;
+
+        /** Whether setting SAML required setting is in progress */
+        isSamlRequiredLoading?: boolean;
+
+        /** Whether setting SAML required setting has failed and why */
+        samlRequiredError?: OnyxCommon.Errors;
+    };
 }>;
 
+/** Model of SAML metadata */
+type SamlMetadata = {
+    /**
+     * The format of the name identifier to be used in SAML assertions
+     * Common formats include 'emailAddress' for email addresses or 'persistent' for persistent pseudonymous identifiers.
+     */
+    nameFormat?: string;
+
+    /**
+     * The unique identifier for the SAML entity (typically a URI)
+     * It is used to identify the issuer of a SAML request or the intended recipient of a SAML response.
+     */
+    entityID?: string;
+
+    /** The URL to which users should be redirected for SAML-based authentication (SAML login endpoint) */
+    urlLogin?: string;
+
+    /**
+     * The URL to which users can be redirected for logging out (SAML logout endpoint)
+     * This is used to initiate the single logout process.
+     */
+    urlLogout?: string;
+
+    /**
+     * SAML metadata of the service provider
+     * This metadata includes keys and endpoints needed to establish the service provider in a SAML exchange.
+     */
+    metaService?: string;
+
+    /**
+     * SAML metadata of the identity provider
+     * This metadata includes keys and endpoints necessary for the identity provider to participate in SAML exchanges.
+     */
+    metaIdentity?: string;
+
+    /** Whether the SAML metadata is currently being fetched */
+    isLoading?: boolean;
+
+    /** Whether fetching the SAML metadata has failed and why */
+    errors: OnyxCommon.Errors;
```

Comment:
@NikkiWines the backend doesn't set any onyxdata errors when getting SAML settings fails, right? (through the GetSAMLSettings command)

Reply:
thanks ^^